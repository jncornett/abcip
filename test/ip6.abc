d ( stack="eth:ip6:hop6:rte6:frag6:dst6:tcp" )

a (
   hop6:opt="| 0 0 0 0 0 0 |",
   rte6:addr="| 0 1 2 3 4 5 6 7 8 9 a b c d e f |",
   frag6:id=0x12345678,
   dst6:opt="| 0 0 0 0 0 0 |",
   tcp:pay="ABC"
)

