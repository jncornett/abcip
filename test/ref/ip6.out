IP6 (flowlabel 0x00001, hlim 64, next-header Options (0) payload length: 71) ::ffff:10.1.2.3 > ::ffff:10.9.8.7: HBH (pad1)(pad1)(pad1)(pad1)(pad1)(pad1)srcrt (len=2, type=0, segleft=0, rsv=0x0, [0]1:203:405:607:809:a0b:c0d:e0f) frag (0x12345678:0|31) DSTOPT (pad1)(pad1)(pad1)(pad1)(pad1)(pad1)48620 > 8: Flags [], seq 0:3, win 8192, length 3
	0x0000:  0209 0807 0605 0201 0203 0405 86dd 6000  ..............`.
	0x0010:  0001 0047 0040 0000 0000 0000 0000 0000  ...G.@..........
	0x0020:  ffff 0a01 0203 0000 0000 0000 0000 0000  ................
	0x0030:  ffff 0a09 0807 2b00 0000 0000 0000 2c02  ......+.......,.
	0x0040:  0000 0000 0000 0001 0203 0405 0607 0809  ................
	0x0050:  0a0b 0c0d 0e0f 3c00 0000 1234 5678 0600  ......<....4Vx..
	0x0060:  0000 0000 0000 bdec 0008 0000 0000 0000  ................
	0x0070:  0000 5000 2000 2f97 0000 4142 43         ..P.../...ABC
